Core/src/org/sleuthkit/autopsy/datamodel/CommonFileNode.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2011-2017 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.datamodel;
+
+import org.sleuthkit.datamodel.AbstractFile;
+
+/**
+ *
+ * @author bsweeney
+ */
+public class CommonFileNode extends AbstractFsContentNode<AbstractFile> {
```

Comment:
This class does not need to extend AbstractFsContentNode<AbstractFile>. That base class is only needed for nodes in the tree view. The nice thing about this is that you will not need to implement the ContentNode and DisplayableItemNode interface methods, which are currently throwing unsupportedOperationException. Instead, you need to provide an override of AbstractFile.createSheet.